pom.xml

```diff
@@ -380,7 +389,21 @@
                   <!-- <bind> <volume>${image.root}:/images/</volume> </bind> -->
                 </volumes>
                 <env>
+                  <HAUTH_VERSION>${project.version}</HAUTH_VERSION>
+                  <!--  Default Docker bridge host IP -->
+                  <DB_HOST>172.17.0.1</DB_HOST>
+                  <DB_PORT>${test.db.port}</DB_PORT>
+                  <DB_PASSWORD>${test.db.password}</DB_PASSWORD>
+                  <DB_CACHE_PORT>${test.db.cache.port}</DB_CACHE_PORT>
+                  <CAMPUS_NETWORK_SUBNETS>172.17.0.0/24,192.168.0.0/24</CAMPUS_NETWORK_SUBNETS>
+                  <SECRET_KEY_GENERATION_PASSWORD>password</SECRET_KEY_GENERATION_PASSWORD>
```

Comment:
I'm a little wary of default passwords and the like. Is there a reason we couldn't use the UUID generating Maven plugin to create on-the-fly UUIDs for this? I know this container is ephemeral (for testing) so this is probably overkill but it still feels better to engrain the behavior.

Cf. https://github.com/ksclarke/freelib-maven-plugins/blob/main/src/main/java/info/freelibrary/maven/UUIDGeneratingMojo.java

Reply:
The same key needs to be generated each time the application is run, right? Otherwise we'd lose the ability to decrypt cookies in the case of e.g. redeploying the application between the time when an access cookie is issued to the client and the time when it is used by the client.

Reply:
Er, wait, do you mean just for testing purposes?

Reply:
Yes, these are the test containers config right? The prod ENV would be set differently.

Reply:
my bad, I misunderstood